Jellyfin.ApiClient/ApiClient.cs

```diff
@@ -338,9 +296,9 @@ public Task<HttpResponse> GetResponse(string url, CancellationToken cancellation
         /// <param name="cancellationToken">The cancellation token.</param>
         /// <returns>Task{Stream}.</returns>
         /// <exception cref="System.ArgumentNullException">url</exception>
-        public Task<Stream> GetImageStreamAsync(string url, CancellationToken cancellationToken = default(CancellationToken))
+        public Task<Stream> GetImageStreamAsync(Uri url, CancellationToken cancellationToken = default)
         {
-            if (string.IsNullOrEmpty(url))
+            if (string.IsNullOrEmpty(url.ToString()))
```

Comment:
Same

Reply:
There are lots of similar checks that worked on a `string url` that aren't necessarily required with a `Uri url`. In order to get the library to a working state I just did things like that to avoid any structural changes. That conditional will never evaluate as true because `new Uri("")` would throw an error. Should probably be `if (url == null)` instead.

Reply:
Well, that's why I'm mentioning it :wink:

Reply:
Fixed in 5ae2f74